File: Calculadora de Notas em Java/src/Usuario.java
import java.util.Scanner;
import javax.swing.*;


public class Usuario {
    public static void main(String[] args) {
        Scanner ler = new Scanner(System.in);
        var continuar = true;
        while (continuar) {
            Matricula conferir = new Matricula();
            Notas notasaluno = new Notas();
            conferir.matriculas();
            notasaluno.notas();
            notasaluno.soma();
            notasaluno.resultado();
            String escolhaString = JOptionPane.showInputDialog("Deseja fazer outra consulta?\nDigite 1 para continuar !\nOu qualquer outra tecla para sair");
            int escolha = Integer.parseInt(escolhaString);

            if (escolha == 1) {
                continuar = true;
            } else {
                JOptionPane.showMessageDialog(null, "Obrigado por usar nosso programa!");
                continuar = false;
            }

        }
    }
}

File: Calculadora de Notas em Kotlin/src/main/kotlin/Usuario.kt
import java.util.Scanner

fun Usuario() {
    val ler = Scanner(System.`in`)
    var continuar = true
    while (continuar) {
        val matricula = matriculas()
        val notas = Notas()
        while (true) {
            print(
                "Deseja fazer outra consulta\n" +
                        "[1] Para sim\n" +
                        "[2] Para sair\n" +
                        "-> "
            )
            var escolha = ler.nextInt()
            if (escolha == 1) {
                continuar = true
                break

            } else if (escolha == 2) {
                println("Obrigado por usar nosso codigo");
                continuar = false
                break

            } else {
                println("Digite uma opcao valida")

            }
        }

    }
}
